### ISSUE TITLE
a11y - Accessibility warning Interactive controls must not be nested is reported by Axe tools.

### ISSUE THREAD

**Issue by izaac** [rancher/dashboard#15953]
**Setup**
- Rancher version: `v2.13.0-alpha7`
- Browser type & version: `Chrome` `142.0.7444.135`

**Describe the bug**

 Interactive controls must not be nested is reported by Axe tools when running an accessibility scan. And flagged as `Serious`

**To Reproduce**

- Install Rancher
- Login
- Run the Axe extension tools and run an Scan.

**Result**

The report shows

<img width="535" height="342" alt="Image" src="https://github.com/user-attachments/assets/23c313bf-2502-4315-9299-2d3d772397d1" />

**Expected Result**
The scanner not able to detect this issue and pass the best practices filter.

Found while QA: https://github.com/rancher/dashboard/issues/15169#issuecomment-3530301447

**Comment by aalves08** [comment]
@izaac these type of violations should have been fixed by https://github.com/rancher/dashboard/pull/18480, which is merged. I think the count was down to 1 again. We plan on tackle the rest of the violations in a dedicated issue. Can we close this?
